core/src/test/java/org/infinispan/stats/SingleStatsTest.java

```diff
@@ -87,8 +87,8 @@ protected void configure(ConfigurationBuilder cfg) {
          memoryConfigurationBuilder.maxCount(size);
       } else {
          if (storageType == StorageType.OFF_HEAP) {
-            // When expiration eviction is used it stores the internal metadata which we will estimate as 24 bytes
-            long amountPerEntry = OFF_HEAP_SIZE + (evictionStrategy.isExceptionBased() ? 24 : 0);
+            // When expiration eviction is used it stores the internal metadata which we will estimate as 12 bytes
+            long amountPerEntry = OFF_HEAP_SIZE + (evictionStrategy.isExceptionBased() ? 12 : 0);
```

Comment:
previously, the `amountPerEntry` was always `OFF_HEAP_SIZE`.

Reply:
I am aware, and that was not correct.

Reply:
We can probably lower OFF_HEAP_SIZE, but the exception based size needs to be accounted for.

Reply:
I have updated it with exact memory allocation sizes now.